site/layouts/_default/docs.html

```diff
@@ -21,8 +21,8 @@ <h5 class="offcanvas-title" id="bdSidebarOffcanvasLabel">Browse docs</h5>
           </a>
           <h1 class="bd-title mb-0" id="content">{{ .Title | markdownify }}</h1>
         </div>
-        <p class="bd-lead">{{ .Page.Params.Description | markdownify }}</p>
-        {{ partial "ads" . }}
```

Comment:
Is that mandatory since `ads.html` is already empty ? Or should we remove the file ?

Reply:
Let's remove this useless `ads.html` file (see https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/pull/1367/commits/329ef1b508e61868cc1fdffcd73ae6d79679da67).